src/app/personalize-menu/personalize-menu.component.ts

```diff
@@ -82,18 +69,29 @@ export class PersonalizeMenuComponent implements OnInit {
     const currentTheme = this.personalize.theme = this.theme;
     const currentColour = this.personalize.colors = this.colour;
 
+    this.themeMenuItems = this.personalize.themes();
+
+    const personalizationColors = this.personalize.personalizationColors();
+    this.colourMenuItems = Object.keys(personalizationColors).map(colorId => personalizationColors[colorId]);
+
+    this.setSelectedTheme(currentTheme);
+    this.setSelectedColor(currentColour);
```

Comment:
not just this line, but it looks like a lot of variation between American English and British English > colour versus color

Reply:
yea, that's how @bthharper build it originally
I think I'll just change back to use American English